dear_petition/petition/constants.py

```diff
@@ -42,3 +42,16 @@
 )
 
 NORTH_CAROLINA = "NC"
+DURHAM_COUNTY = "DURHAM"
+
+DISTRICT_COURT_WITHOUT_DA_LEAVE = "Dismissal without Leave by DA"
+
+DISPOSITION_METHODS = (
```

Comment:
If we're moving this to constants, I think we should prepend with dismissed:

```suggestion
DISMISSED_DISPOSITION_METHODS = (
```

Reply:
done